package.json

```diff
@@ -5,10 +5,11 @@
   "homepage": "https://nodejs.org",
   "scripts": {
     "build": "node build.js",
+    "build:deploy":"node build.js --preserveLocale",
     "serve": "node server.js",
```

Comment:
Since we also have `preserveLocale` in `server.js`, is it more reasonable to serve as `node server.js --preserveLocale` ?

Reply:
I think while developing is less needed to build all the files for every language. I think it's more relevant when deploying. But a second script in package.json might help.

Current options to serve with preserveLocale are:
```
npm start -- -- --preserveLocale
npm run serve -- --preserveLocale
node server.js --preserveLocale
```

I could add a script to run:
```
npm run serve:preserveLocale
```